disdrodb/L0/readers/TUTORIAL/TUTORIAL_correction.py

```diff
@@ -17,77 +17,10 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 # -----------------------------------------------------------------------------.
 
-import os
-import sys
-import click
 from disdrodb.L0.L0_processing import run_L0
 
```

Comment:
This reader and directory should be removed in my opinion !

Reply:
ok @ghiggi , but it is the last part of the tutorial (notebook).

Reply:
True. We will need to update the notebook. 
And the section "Step 3 : Create the reader" should directly point the user to add the reader to the appropriate directory `DATA_SOURCE/CAMPAIGN_NAME`. 
It will not be possible anymore to launch the script in the described way.
But instead run `run_disdrodb_reader ... ` ;)